src/cgame/cg_ents.cpp

```diff
@@ -748,7 +748,8 @@ static void CG_LightFlare( centity_t *cent )
 	}
 
 	//can only see the flare when in front of it
-	flare.radius = len / es->origin2[ 0 ];
+	// Avoid division-by-zero
+	flare.radius = es->origin2[ 0 ] ? len / es->origin2[ 0 ] : len;
```

Comment:
Apparently this comes from a `radius` keyword on the entity which is stored into the `activatedPosition` field. A cleaner fix would to be to check this in SP_gfx_light_flare, emit a warning and either delete the entity or set a default radius.